bpf/process/bpf_generic_kprobe.c

```diff
@@ -229,51 +230,15 @@ generic_kprobe_process_filter(void *ctx)
 // see also: MIN_FILTER_TAILCALL, MAX_FILTER_TAILCALL
```

Comment:
Jiri I would just remove the comment above about kprobe indexes or update it... also probably same for MIN_FILTER_TAILCALL  or remove it.

Reply:
ah, right, will remove that